src/test/ChipsDao.test.ts

```diff
@@ -0,0 +1,60 @@
+import { expect } from 'chai';
+import * as sinon from 'sinon';
+import * as oracledb from 'oracledb';
+import ChipsDao from '../daos/CHIPS/ChipsDao';
+
+const config = {
+    user: "USER",
+    password: "PASSWORD",
+    connectionString: "CONSTRING"
+};
+
+const DATA = {
+    "XXXX1234": { transaction_id: "123456789", orgunit_desc: "ORG UNIT DESC" }
+}
+
+const CORRECT_BARCODE = "XXXX1234";
+const INCORRECT_BARCODE = "YYYY1234";
+
+const chipsDao = new ChipsDao();
+const error = new Error('test error');
+  
+describe('Chips database call', () => {
+    before( async ()=> {
+        sinon.stub(oracledb, 'getConnection').resolves({
+            execute: function() {},
+            close: function() {}
+        });
+        let connection = await oracledb.getConnection(config);
+        connection.execute = sinon.stub();
+        connection.execute.withArgs(CORRECT_BARCODE, sinon.match.any).resolves({
+            rows: {"XXXX1234": { transaction_id: "123456789", orgunit_desc: "ORG UNIT DESC" }}
+        });
+    })
+    after(()=> {
+        oracledb.getConnection.restore();
+    })
+    it('test makeQuery is successfully called', ()=>{
```

Comment:
I may be wrong but this test feels a little unnecessary. You wouldn't normally just test a function being called in java.